ci/build/build-ray-docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -exuo pipefail
+
+WHEEL_NAME="$1"
+SOURCE_IMAGE="$2"
+CONSTRAINTS_FILE="$3"
+DEST_IMAGE="$4"
+
+CPU_TMP="$(mktemp -d)"
+
+mkdir -p "${CPU_TMP}/.whl"
+
+cp .whl/"$WHEEL_NAME" "${CPU_TMP}/.whl/${WHEEL_NAME}"
+cp docker/ray/Dockerfile "${CPU_TMP}/Dockerfile"
+cp python/requirements_compiled.txt "${CPU_TMP}/."
+cp python/requirements_compiled_py37.txt "${CPU_TMP}/."
+
+cd "${CPU_TMP}"
+tar --mtime="UTC 2020-01-01" -c -f - . \
+    | docker build --progress=plain \
+        --build-arg FROM="$SOURCE_IMAGE" \
+        --build-arg WHEEL_PATH=".whl/${WHEEL_NAME}" \
+        --build-arg CONSTRAINTS_FILE="$CONSTRAINTS_FILE" \
+        -t "$DEST_IMAGE" -f Dockerfile -
```

Comment:
should we use wanda here? so you do not need to `tar` command, because wanda will handle that.

Reply:
possibly, but where is a follow up logic afterwards for generating aliases and publishing docker image which happens outside of wanda so it needs a subsequent buildkite job, etc; or wanda needs to support a post-command feature, both seems complex

also the resulting docker image will always be different so less cache benefit from wanda

Reply:
you can use wanda just as a stand-alone command to build an image.

Reply:
it does not have to be used with rayci.